caravel/const.py

```diff
@@ -0,0 +1,15 @@
+""" Package constants """
+
+from looper import __version__ as looper_version
+from divvy.const import COMPUTE_SETTINGS_VARNAME
+LOOPER_VERSION = looper_version
+REQUIRED_LOOPER_VERSION = "0.11.0"
```

Comment:
Can we bound our `looper` req in the requirements file, then read this from there?

Reply:
yeah, good point